### DISCUSSION TITLE
How to know the current slide from the carousel component

### DISCUSSION THREAD

**Discussion by jaads** [saadeghi/daisyui#701] [Q&A] [answered]
Hey I want my users to know on which slide they are when using the carousel component. Any ideas on how I can do this? 
Here is an example how I would like to have it:

![image](https://user-images.githubusercontent.com/34009329/162759904-70f241c4-d21e-4f34-93b2-06ea7dd6394e.png)

I currently created circles below the carousel but what I'm missing is, that the circle representing the current image is highlighted. At best I would find a solution without using anchor links. The users don't need to be able to click on certain circle.

Thanks in advance!

**Comment by saadeghi** [comment, marked as answer]
I would suggest using a JS image slider for more functionality. daisyUI is just a scrollable area with scroll snap and there are limits with a pure CSS image slider.
